edx-platform/utec/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}
+                    You have been enrolled in {{ course_name }}
+                {% endblocktrans %}
+            </h1>
+            
+            <p style="color: rgba(0,0,0,.75);">
+                Hola {{ full_name }},
+                <br />
+            </p>
+
+            <p style="color: rgba(0,0,0,.75);">
+                Has sido inscripto por invitación en el curso “{{ course_name }}”. El curso estará visible una vez que accedas, en el panel principal de la plataforma EDU con tu usuario.
```

Comment:
typo: inscripto

Reply:
not a typo, that is how they say it in some places @jfavellar90 it was like that in UTEC requirements